### PR TITLE
Bp 0207 testing product view

### PR DESCRIPTION
**Pull request by friedow**

### CODE REVIEW

File: src/__tests__/ProductView/EventTable.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import EventTable from '../../ProductView/EventTable/EventTable.js';
+import renderer from 'react-test-renderer';
+import TestEventTable from './../testData/eventTable.js';
+let instance;
+
```
Comment (julianweise) [inline src/__tests__/ProductView/EventTable.js:6]: Maybe it would be reasonable to also test the functions testFilterArguments and testFilter explicitly in order to cover edge cases or get a more sufficient feedback in case of an error.
What do you think?

File: src/__tests__/ProductView/ProductView.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import TestEventTypes from './../testData/eventTypes.js'
+import TestEvents from './../testData/events.js'
+let instance;
+
+test("Rendering of ProductView", () => {
+    const mockCallback = jest.fn();
+    const component = renderer.create(
+        <ProductView
+            ref={(child) => {instance = child}}
+            dataSource={{fetchProduct: mockCallback, fetchEventTypesOf: mockCallback, fetchEventsOf: mockCallback}}
```
Comment (julianweise) [inline src/__tests__/ProductView/ProductView.js]: Please add line breaks after each element of the dataSource object.

File: src/__tests__/ProductView/ProductView.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            params={{productId: 0}}/>
+    );
+    instance.handleProductData(TestProduct.PRODUCT);
```
Comment (julianweise) [inline src/__tests__/ProductView/ProductView.js]: Nice one using mockCallback!
But I would suggest to generate an independent test for each of the functions.

**Review by julianweise — CHANGES_REQUESTED** [review]
Nice tests! Only suggested some small code refactorings.

**Review by julianweise — APPROVED** [review]
